### PR TITLE
HLD to reduce disk writes on SONiC images

### PR DESCRIPTION
**Pull request by assrinivasan**
This HLD analyzes disk write patterns on Arista devices running various SONiC OS versions, focusing on reducing excessive writes that lead to storage degradation.

| Repo | PR Title |  State  |
|----------|:-------------:|------:|
|sonic-buildimage| Modify bgpmon to write vtysh cmd output to /dev/null| N/A
|sonic-buildimage| Move monit and logrotate files to tmpfs | N/A
|sonic-buildimage| Disable Kernel Journaling on SONiC OS| N/A
|sonic-swss|Move supervisor logs to tmpfs| N/A

**Disable Kernel Journaling on SONiC OS** PR subject to acceptance from community and MSSONiC.

### CODE REVIEW

File: doc/sonic-reduce-disk-io/sonic-reduce-disk-io.md
Comment (prgeor) [inline doc/sonic-reduce-disk-io/sonic-reduce-disk-io.md:1]: @assrinivasan can you list the cons of disabling journaling for ext4?

**Comment by prgeor** [comment]
@assrinivasan is it ready for review? still in draft?

**Comment by assrinivasan** [comment]
> @assrinivasan is it ready for review? still in draft?

Ready for revise Prince.

File: doc/sonic-reduce-disk-io/sonic-reduce-disk-io.md
Comment (assrinivasan) [reply doc/sonic-reduce-disk-io/sonic-reduce-disk-io.md:1]: Added in latest Prince.

**Pull request closed without merging**